shipanaro/models.py

```diff
@@ -6,25 +6,30 @@
 # Based on ISO 5218
 SEXES = (
     # (0, 'Not known'),
-    (1, _('Male')),
-    (2, _('Female')),
-    (9, _('Not Applicable')),
+    (1, _("Male")),
+    (2, _("Female")),
+    (9, _("Not Applicable")),
 )
 
 # Based on ISO 5218 and Best Practices for Asking Questions to Identify
 # Transgender and Other Gender Minority Respondents on Population-Based
 # Surveys.
 GENDERS = (
-    (1, _('Male')),
-    (2, _('Female')),
-    (9, _('Gender non-conforming')),
+    (1, _("Male")),
+    (2, _("Female")),
+    (3, _("Agender")),
+    (4, _("Fluid")),
+    (5, _("Non-binary")),
+    (6, _("Pangender")),
+    (7, _("Transgender")),
+    (9, _("Gender non-conforming")),
```

Comment:
Extended to match current applicant form in https://participa.pirates.cat/form/inscripcio